composer.json

```diff
@@ -67,7 +71,8 @@
 
 		"symfony/browser-kit": "~2.7",
 		"silex/web-profiler": "~2.0",
-		"sorien/silex-dbal-profiler": "~2.0"
+		"sorien/silex-dbal-profiler": "~2.0",
+		"wmde/fundraising-frontend-content": "dev-master as 1.0.x-dev"
```

Comment:
Cool, did not know this stuff. https://getcomposer.org/doc/articles/aliases.md#require-inline-alias

What is the advantage of having it here?